members/D000622.yaml

```diff
@@ -126,11 +126,11 @@ contact_form:
           value: $MESSAGE
           required: true
     - javascript:
-        - name: char replace
-          values: ["$MESSAGE"]
-          selectors: ["textarea#input-356E5E00-5056-A066-603C-9FAC0F3B6888"]
-          commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/(%2[0C])/g, '').replace(/(CC:)/g,'CC');}"]
```

Comment:
why'd you remove the replacement for `CC:` ?

Reply:
this one is fixed